src/xunit.netcore.extensions/Attributes/ActiveIssueAttribute.cs

```diff
@@ -11,7 +11,7 @@ namespace Xunit
     /// Apply this attribute to your test method to specify an active issue.
     /// </summary>
     [TraitDiscoverer("Xunit.NetCore.Extensions.ActiveIssueDiscoverer", "Xunit.NetCore.Extensions")]
-    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class, AllowMultiple = true)]
```

Comment:
Yes. I've tested locally. Since this is handled by Xunit it gets all the attributes in a member (either class or method) and it calls the discoverer or GetTraits method for each of them and add those traits to each guy. So yes xunit will call out GetTraits multiple time with each AttributeInfo and then we will return the corresponding traits for each Attribute.